ipv-stub/ipv-stub-requests.http

```diff
@@ -0,0 +1,13 @@
+### AUTHORIZE
+GET https://ipvstub.signin.dev.account.gov.uk/authorize
```

Comment:
Just wondering, what's the value of having this one? It won't work without a generated request - do we want to maintain a hard-coded request in here that works with the current validation and decryption?

Reply:
May be possible to import the script. I see IDEA has an example

```
### Composing scripts with imports
< {%
    // common script parts can be moved to separate files and imported with 'import' keyword
    import {makeSignature} from "./my-utils";

    request.variables.set("signature", makeSignature())
%}
POST https://examples.http-client.intellij.net/post
X-My-Signature: {{signature}}
Content-Type: application/json
```